libraries/MySensors/MySensor.cpp

```diff
@@ -102,7 +102,7 @@ void MySensor::begin(void (*_msgCallback)(const MyMessage &), uint8_t _nodeId, b
 
 	// Setup radio
 	if (!radio.init()) {
-		debug(PSTR("radio initialization failed!"));
+		debug(PSTR("radio initialization failed!\n"));
```

Comment:
No sure it is worth continuing here if radio initialisation fails. 

How about just do a while(1); ?